pyproject.toml

```diff
@@ -4,7 +4,8 @@ build-backend = "setuptools.build_meta"
 
 [project]
 name = "python-can"
-dynamic = ["readme", "version"]
+dynamic = ["readme"]
+version = "4.3.1"
```

Comment:
Please undo this. The version is set by setuptools-scm from the git tag.

Reply:
This is now undone. Please accept if no other problems. Thank you

Reply:
@pratik-mahamuni the tests are failing

Reply:
@zariiii9003 modified the method `equals` from the `Message` class based on the intricacies of how the tests are modeled in the repository. This change could be deemed temporary as the current implementation of the loggers_test.py uses the same data and constructor for all file types (ASC, BLF, CSV etc.).

As `equals` is used only in tests and not for logic in the code (based on a quick `grep`), the small change should not have side-effects. I will let you be the judge for that.